.github/workflows/BuildDockerContainer.yaml

```diff
@@ -52,7 +51,7 @@ jobs:
           file: "./containers/Dockerfile.buildenv"
           target: dev
           tags: ${{ inputs.image_name }}:dev
-          platforms: linux/amd64 #,linux/arm64
+          platforms: linux/amd64,linux/arm64
```

Comment:
Can we build the container on 2 cores by setting `PARALLEL_MAKE_ARG=-j2` somehow?

Reply:
I think it already builds on 2 cores because that's the default in the container

Reply:
Ah yes you are right.